### PR TITLE
Niloofar/ Fix Tooltip Component Issues

### PR DESCRIPTION
**Pull request by niloofar-deriv**
This PR contains temporary fix of this raised issue:

https://github.com/deriv-com/p2p/assets/93518187/5f764bd3-14eb-4914-b5db-3f1f9be537a8

### CODE REVIEW

File: src/components/TooltipMenuIcon/TooltipMenuIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+type AsElement = 'a' | 'button' | 'div';
+type TTooltipMenuIcon<T extends AsElement> = ComponentProps<T> & {
+    as: T;
+    disableHover?: boolean;
+    tooltipContent: string;
+    tooltipPosition?: TooltipProps['placement'];
+};
+
+const TooltipMenuIcon = <T extends AsElement>({
```
Comment (ameerul-deriv) [inline src/components/TooltipMenuIcon/TooltipMenuIcon.tsx:20]: nit: can you add a todo here to replace this with the component that will eventually come from deriv ui

File: package.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     },
     "dependencies": {
         "@babel/preset-env": "^7.24.5",
+        "@chakra-ui/react": "^2.8.2",
```
Comment (ameerul-deriv) [inline package.json:18]: in deriv ui, will you also use charkra ui 👀

Comment (niloofar-deriv) [reply package.json:18]: No it's a temporary thing and a quick fix

Comment (farrah-deriv) [reply package.json:18]: @niloofar-deriv can we have the fix in deriv-ui instead of installing this dependency?

Comment (niloofar-deriv) [reply package.json:18]: Base on my discussion with @matin-deriv @ali-hosseini-deriv we decided to unblock the release for now in parallel I'm working on fixing this issue from ui side as well

**Review by ameerul-deriv — APPROVED** [review]

**Review by nada-deriv — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (shayan-deriv) [reply package.json:18]: @farrah-deriv we will make sure to keep the api the same when we fix it later in ui so there won't be any changes from ur side other than changing the import path.

**Pull request merged**